docs/example_compartmental_elongated_cells_tutorial.rst

```diff
@@ -0,0 +1,805 @@
+            <Parameters TypeName="Top" MaxNumberOfJunctions="1" NeighborOrder="1"/>
+            <Parameters TypeName="Center" MaxNumberOfJunctions="2" NeighborOrder="1"/>
+        </InternalTypeSpecificParameters>
+
+    </Plugin>
+
+
+With this extra addition the compartments will form a line even if we let the simulation run for a very long time:
+
+|img013|
+
+
+As you probably have noticed, the syntax for this plugin resembles the syntax of the FPP plugin - we have ``<Lambda>``,
+``<ActivationEnergy>``, ``MaxNumberOfJunctions`` . This is because Curvature plugin establishes its own set of "links" between
+cells but those links are not used to penalize intercellular distance but rather to penalize the deviation from straight line
+arrangement of compartment cells
+
+Adding persistent motion to cells
++++++++++++++++++++++++++++++++++
+
+Let us add a bit more code to make this simulation more interesting. First, we will create more cells. We will use
+our convenience function ``create_arranged_cells`` and as a result all of those cells will be arranged vertically -
+this will not be a problem though because, next, we will be applying random force to the "first" cell of each cluster i.e.
+to the cell that is created first in each cluster. We will store a list of "first" cells inside member variable
+``self.list_of_leading_cells = []`` which is a list. . we have to be careful to ensure that cells stored in that list do
+not disappear because if the do disappear and we try to reference them we will get Segmentation Fault Error.
+We will show later how we could avoid this issue in the code , just to show you how to handle situation of that type.
+Before we apply any force, we will give simulation a generous 300 MCS for all the FPP links to get established.
+If we applied force before links are established it is likely that some ``Top`` cell could have moved away from the
+cluster before links had a chance to form. Next, every 500 MCS we will reassign random forces applied to "first" cells.
+
+The simulation code can be found in ``Demos/CompuCellPythonTutorial/ElongatedCellsTutorial/Tutorial_09``
+
+
+In terms of XML modification, we only need to add a one-liner that enavbles ExternalPotential plugin that simulates
+external force:
+
+.. code-block:: XML
+
+    <Plugin Name="ExternalPotential"/>
+
+Notice that we do not specify any parameters because we will use Python to set force vectors applied to "first" cells
+
+The ``ElongatedCellsSteppables.py`` is more interesting:
+
+.. code-block:: Python
+
+    from cc3d.core.PySteppables import *
+    import random
+
+
+    class ElongatedCellsSteppable(SteppableBasePy):
+        def __init__(self, frequency=1):
+
+            SteppableBasePy.__init__(self, frequency)
+            self.list_of_leading_cells = []
+            self.maxAbsLambdaX = 10
+
+        def start(self):
+            self.create_arranged_cells(x_s=25, y_s=25, size=5, cell_type_ids=[1, 2, 2, 2, 2, 1])
```

Comment:
You could have a comment to explain why the cells are being created in this particular pattern.

Reply:
I added it in different place